### DISCUSSION TITLE
How to remove the flight control system

### DISCUSSION THREAD

**Discussion by memy009** [JSBSim-Team/jsbsim#925] [Q&A]
Hello everyone, I want to make the control for a 737 (flight control for the horizontal part) and I want to retrieve the status of the uncontrolled system. I know that under JSBSim, the flight controlers are already implemented and so I wanted to know if any of you know how to remove the autopilot. THANKS .

**Comment by seanmcleod** [comment]
The 737 model in the JSBSim repo has no autopilot or any sort of FBW FCS with command laws etc. Take a look at the FCS section of the 737 model.

**Reply by memy009** [reply]
Ok, thank U.

**Comment by memy009** [comment]
So these are not an implementation of a controlleur ??? 

<flight_control name="FCS: 737">



        <channel name="Pitch">



            <summer name="Pitch Trim Sum">

                <input>fcs/elevator-cmd-norm</input>

                <input>fcs/pitch-trim-cmd-norm</input>

                <clipto>

                    <min>-1</min>

                    <max> 1</max>

                </clipto>

            </summer>



            <aerosurface_scale name="Elevator Control">

                <input>fcs/pitch-trim-sum</input>

                <range>

                    <min>-0.3</min>

                    <max> 0.3</max>

                </range>

                <output>fcs/elevator-pos-rad</output>

            </aerosurface_scale>



            <aerosurface_scale name="Elevator Normalized">

                <input>fcs/elevator-pos-rad</input>

                <domain>

                    <min>-0.3</min>

                    <max> 0.3</max>

                </domain>

                <range>

                    <min>-1</min>

                    <max> 1</max>

                </range>

                <output>fcs/elevator-pos-norm</output>

            </aerosurface_scale>



        </channel>

        <channel name="Roll">



            <summer name="Roll Trim Sum">

                <input>fcs/aileron-cmd-norm</input>

                <input>fcs/roll-trim-cmd-norm</input>

                <clipto>

                    <min>-1</min>

                    <max> 1</max>

                </clipto>

            </summer>



            <aerosurface_scale name="Left Aileron Control">

                <input>fcs/roll-trim-sum</input>

                <range>

                    <min>-0.35</min>

                    <max> 0.35</max>

                </range>

                <output>fcs/left-aileron-pos-rad</output>

            </aerosurface_scale>



            <aerosurface_scale name="Right Aileron Control">

                <input>-fcs/roll-trim-sum</input>

                <range>

                    <min>-0.35</min>

                    <max> 0.35</max>

                </range>

                <output>fcs/right-aileron-pos-rad</output>

            </aerosurface_scale>



            <aerosurface_scale name="Left aileron Normalized">

                <input>fcs/left-aileron-pos-rad</input>

                <domain>

                    <min>-0.35</min>

                    <max> 0.35</max>

                </domain>

                <range>

                    <min>-1</min>

                    <max> 1</max>

                </range>

                <output>fcs/left-aileron-pos-norm</output>

            </aerosurface_scale>

**Comment by seanmcleod** [comment]
You asked:

> how to remove the autopilot.

I said:

> The 737 model in the JSBSim repo has no **autopilot** or any sort of **FBW FCS** with command laws etc. 

Do you see any implementation of an **autopilot** or **FBW FCS** in the 737's FCS section?

It's purely a direct pilot input control system. The pilot inputs their input via `fcs/elevator-cmd-norm` and `fcs/pitch-trim-cmd-norm` which are in the range `[-1, +1]` and which are then simply scaled to an angular deflection of between `[-0.3, +0.3]` radians, using the longitudinal input to elevator position as an example. The same applies for the lateral and directional inputs.

**Reply by memy009** [reply]
Endeed, I don't see them ... But I don't understand what did the snapshots means so I asked ... Thank's for the naswer !!!
